app/Http/Controllers/Api/v1/CampaignMemberApiController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Controllers\Api\v1;
+
+use App\Http\Resources\CampaignUserResource;
+use App\Models\Campaign;
+use App\Models\CampaignUser;
+
+class CampaignMemberApiController extends ApiController
+{
+    public function index(Campaign $campaign)
+    {
+        $this->authorize('access', $campaign);
+
+        return CampaignUserResource::collection($campaign->members);
+    }
+
+    public function show(Campaign $campaign, CampaignUser $campaignUser)
+    {
+        $this->authorize('access', $campaign);
+
+        return new CampaignUserResource($campaignUser);
+    }
```

Comment:
Hey, before going any further I wanted to check that I could correctly fetch the CampaignUser but while using this I only have an empty resource:
```
{
    "data": {
        "id": null,
        "roles": [],
        "user": null
    }
}
```
I'm probably missing something since this is a Pivot resource.
Any suggestions ?